src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,62 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+This version comes with a breaking change. The default ingress class for ingresses managed by okteto is `okteto-nginx` instead of `nginx` to prevent colissions with other controllers that might be already installed in the cluster. All ingresses managed by Okteto will be automatically updated. No action is required from the user. If you want to keep the previous value (`nginx`), you would need to set the following values in your helm values file:
```

Comment:
We use a header for breaking changes, check the older release notes, I think we've done it before.

Reply:
In `1.0.0` we didn't include it but in a previous one we did so, so I wasn't sure about it. As the most recent one was `1.0.0`, I didn't include it but I think it's better to include it, I'll do it.